lib/router/router.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:fluro/fluro.dart';
+import 'package:flutter/widgets.dart';
+import 'package:superformula_test/pages/home_page.dart';
+import 'package:superformula_test/pages/qr_page.dart';
+import 'package:superformula_test/pages/qr_scanner.dart';
+
+class AppRoutes {
+  static final router = FluroRouter();
+
+  static Future<dynamic> navigate(BuildContext context, String routeName,
+      {dynamic args}) {
+    return router.navigateTo(context, routeName,
+        routeSettings: RouteSettings(
+          arguments: args,
+        ));
+  }
+
+  static void createRoutes() {
+    router.define('home', handler: Handler(handlerFunc: (context, params) {
+      dynamic args = context!.settings!.arguments;
+      return MyHomePage(title: args);
+    }));
+    router.define('/qrcode', handler: Handler(handlerFunc: (context, params) {
+      dynamic args = context!.settings!.arguments;
+      return QrCodePage(title: args);
+    }));
+    router.define('/qrscanner',
+        handler: Handler(handlerFunc: (context, params) {
+      dynamic args = context!.settings!.arguments;
+      return QrScannerPage(title: args);
```

Comment:
suggestion: I would cast these as strings instead of using dynamic per the [Effect Dart guidelines](https://dart.dev/guides/language/effective-dart/design#avoid-using-dynamic-unless-you-want-to-disable-static-checking).

Reply:
You know, I did it and then removed it because I figured it would be a question of how it's done the review meeting. Yes, I was prepping for the review meeting lol